src/NewMatcher/dp_config.yaml

```diff
@@ -0,0 +1,29 @@
+bgc_module_remove_score: -3.0
+
+nrp_mon_remove_score: {arg: -2.0, leu: -2.0, val: -2.0, unk: -3.0}
```

Comment:
How about `bgc_module_skip_score` and `nrp_monomer_skip_score`? I like it more because I would like to use the same terminology in Nerpa-MS and there removal/deleton and insertion refer to actual changes in the structure.

Reply:
There's no problem changing it all over the code.

Reply:
I also like `bgc_module_skip_score` and `nrp_monomer_skip_score`, quite clear what this means!